css/styles.css

```diff
@@ -0,0 +1,266 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+    float: center center;
+    background-color: lightgrey;
+    font-family: Merriweather;
+  }
+
+img.background {
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 100%;
+    float: center;
+  }
+
+header  {
+  color: white;
+  background-color: #000000;
+  height: 100px;
+  border: 1px;
+  margin: 0px;
+}
+
+a {
+  color: white;
+}
+
+a:hover {
+  color: #6A5ACD;
+}
+
+h1  {
+  color: yellow;
+  padding-left: 30px;
+  padding-bottom: 5px;
+  font-size: 35px;
+  margin: 1px 8px 1px 8px;
+}
+
+h2  {
+  height: 50px;
+  color: #000000;
+  text-align: center;
+  margin: 5px;
+}
+
+.nav-items {
+  color: white;
+  width: 1020px;
+  padding: 1px 10px 1px 10px;
+  text-align: center;
+  font-size: large;
+  margin:1px 8px 1px 34px;
+  word-spacing: 2px;
+  text-decoration: none;
+}
+
+nav span  {
+  text-transform: uppercase;
+  font-weight: bold;
+  text-decoration: none;
+}
+
+ .taste  {
+  color: grey;
+  text-decoration: line-through;
+}
+
+.are  {
+  color: black;
+  text-decoration: underline yellow;
+
+}
+
+section  {
+  height: 90px;
+  background-color: lightgrey;
+  margin: 0px;
+  padding: 20px;
+  text-align: center;
+  font-size: 35px;
+  border-top: solid yellow;
+}
+
+.head  {
+  background-color: lightgrey;
+  text-align: left;
+  opacity: 0.6;
+  border: 1px solid #333;
+  height: 420px;
```

Comment:
@Schultisal remove the height, try to have the content dictate the height of the component. Then use margin and padding to move and give the inner elements spacing 🔨